galsim/bandpass.py

```diff
@@ -458,17 +458,23 @@ def thin(self, rel_err=1.e-4, preserve_range=False):
 
         @param rel_err            The relative error allowed in the integral over the throughput
                                   function. [default: 1.e-4]
+        @param trim_leading_zeros Remove redundant leading and trailing points where f=0?  (The last
```

Comment:
Maybe remove "leading" from this keyword?  Since it's both leading and trailing zeros.  And `trim_leading_and_trailing_zeros` seems too verbose, even by my standards.